_posts/august-2024-newsletter.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "August Newsletter"
+excerpt: "Our August newsletter highlights key events, including our General Meeting on August 18 and upcoming public meetings at the DART Board and Dallas City Council."
+coverImage: "/assets/blog/august-2024-newsletter/cover.jpg"
+date: "2024-08-08T05:00:00.000Z"
+author:
+  name: DATA Newsletter
+  picture: "/assets/blog/authors/data-team.png"
+ogImage:
+  url: "/assets/blog/august-2024-newsletter/cover.jpg"
+---
+Hello there,  
+
+We have three upcoming events that I want you to be aware of this month.  
+
+## August General Meeting
+
+---
+First, our August General Meeting will be on August 18, 2024 from 3pm to 5pm at the Metropolitan [1200 Main Street, Dallas](https://maps.app.goo.gl/pPBSUsQeJYZr1hxf7) (Enter via Murphy Drive) on the 6th floor, same room as the first meeting we held. We will have light snacks and refreshments available.   
+
+We will be discussing our strategy for two upcoming public meetings that we want to show up in large numbers for.
+ 
+## DART Board Meeting
+
+---
+
+The first meeting is the DART Board meeting at [DART Headquarters](https://maps.app.goo.gl/36uCk3suvRdJPfM47)on August 27, 2024 at 6pm, please arrive 30 minutes early to register for public comments.
+
+Some members of the DART board are not very supportive of transit. Right now, there is a push among some board members to cut DART’s operating budget by 5-10% this year, and cap DART’s revenue growth. Other DART board members have expressed willingness to compromise with these cities that want to cut DART’s funding.
+
+We are here to to communicate 3 things
+
+1. More frequency/service improvements in next year's budget.
+2. Our opposition to these draconian budget cut resolutions, and our disappointment with the member cities who passed them
+3. Our opposition to ANY revenue capping.
+4. Express how DART should oppose any state legislation that cuts/caps their budget
```

Comment:
I think opposing cuts should be 1. rather than more service

Reply:
Agreed